README.md

```diff
@@ -36,72 +38,117 @@
 - [Contributing](#contributing)
 - [License](#license)
 
-
 ## About Mitt Helsingborg
 
-
 ### Built With
 
-* [React Native](https://reactnative.dev/)
-* [TypeScript](https://www.typescriptlang.org/)
-* [Styled Components](https://styled-components.com/)
-
+- [React Native](https://reactnative.dev/)
+- [TypeScript](https://www.typescriptlang.org/)
+- [Styled Components](https://styled-components.com/)
 
 ## Getting Started
 
 To get a local copy up and running follow these simple steps.
 
-
-
 ### Prerequisites
 
 - [Node.js and NPM](https://nodejs.org/en/download/package-manager/)
 - [React Native Development Environment](https://reactnative.dev/docs/environment-setup/)
 - [Eslint](https://helsingborg-stad.github.io/dev-guide/docs/development/linters/eslint.html)
 
-
 ### Installation
 
 1. Clone the repo
+
 ```sh
 git clone https://github.com/helsingborg-stad/app-mitt-helsingborg.git
 ```
+
 2. Install NPM packages
+
 ```sh
 npm install
 ```
+
 3. Run app on iOS simulator
+
 ```sh
 react-native run-ios
 ```
+
 4. Run app on Android emulator
+
 ```sh
 react-native run-android
 ```
 
-#### Component library (Storybook)
-1) Set env variabel IS_STORYBOOK to true in .env
-2) Launch storybook server by running command "npm run storybook" or "yarn storybook"
-3) Launch application in simulator by running command "react-native run-ios".
-4) Now you should se storybook running in the simulator.
+## Setup environment variables
+
+### Backend selector
+
+Using the file .env in the root of the project, various parameters could be used to
+change the behaviour of the app. One of these options is the possibility of building the
+app with an interactive backend picker. This feature offers a dropdown available on the
+login screen that allows the user to choose amongst a number of predefined backends in
+order to troubleshoot or test.
 
+To configure this feature, first put the app build into development mode by adding
+the API_ENV variable to the settings file. Like so:
+
+```
+APP_ENV=development
+```
 
+Next you need to specify the displayname of each backend with the API_ENVS parameter.
+The below list offers four different targets that will be listed in the app in order of
+appearance:
+
+```
+API_ENVS=SANDBOX,DEVELOP,RELEASE,PRODUCTION
+```
+
+NOTE that if the APP_ENV variable is set to production, only the first environment in
+the API_ENVS list will be used and no picker will be available in the app.
+
+For each backend, add two additional keys representing the URL and the APIkey of each:
+
+```
+{NAME}_MITTHELSINGBORG_IO=***
+{NAME}_MITTHELSINGBORG_IO_APIKEY=***
+```
+
+For example:
+
+```
+SANDBOX_MITTHELSINGBORG_IO=***
+SANDBOX_MITTHELSINGBORG_IO_APIKEY=***
+```
+
+#### Component library (Storybook)
+
+1. Set env variabel IS_STORYBOOK to true in .env
+2. Launch storybook server by running command "npm run storybook" or "yarn storybook"
+3. Launch application in simulator by running command "react-native run-ios".
```

Comment:
Launching the application can also be done using `yarn ios` or `yarn android`. They are added as scripts in the package.json

Reply:
I have not modified any documentation related to that, only the Environment picker